closed/src/java.base/share/classes/openj9/internal/security/RestrictedSecurity.java

```diff
@@ -90,8 +90,8 @@ public final class RestrictedSecurity {
         supportedPlatformsNSS.put("Arch", List.of("amd64", "ppc64le", "s390x"));
         supportedPlatformsNSS.put("OS", List.of("Linux"));
 
-        supportedPlatformsOpenJCEPlus.put("Arch", List.of("amd64", "ppc64", "s390x"));
-        supportedPlatformsOpenJCEPlus.put("OS", List.of("Linux", "AIX", "Windows"));
+        supportedPlatformsOpenJCEPlus.put("Arch", List.of("amd64", "ppc64", "s390x", "aarch64"));
+        supportedPlatformsOpenJCEPlus.put("OS", List.of("Linux", "AIX", "Windows", "Mac"));
```

Comment:
The maps `supportedPlatformsNSS` and `supportedPlatformsOpenJCEPlus` need not be captured in fields; they could be local to the static initializer, or, perhaps better, simply listed as cases of switch statements. Also, I think "Mac" should be "Mac OS X" (see [`omrsysinfo_get_OS_type()`](https://github.com/eclipse-omr/omr/blob/master/port/unix/omrsysinfo.c#L2273)).
```java
        boolean isOsSupported;
        boolean isArchSupported;

        // Check whether the NSS FIPS solution is supported.
        switch (props[2]) { // os.name
        case "Linux":
            isOsSupported = true;
            break;
        default:
            isOsSupported = false;
            break;
        }

        switch (props[3]) { // os.arch
        case "amd64":
        case "ppc64le":
        case "s390x":
            isArchSupported = true;
            break;
        default:
            isArchSupported = false;
            break;
        }
        isNSSSupported = isOsSupported && isArchSupported;

        // Check whether the OpenJCEPlus FIPS solution is supported.
        switch (props[2]) { // os.name
        case "AIX":
        case "Linux":
        case "Mac OS X":
        case "Windows":
            isOsSupported = true;
            break;
        default:
            isOsSupported = false;
            break;
        }

        switch (props[3]) { // os.arch
        case "aarch64":
        case "amd64":
        case "ppc64le":
        case "s390x":
            isArchSupported = true;
            break;
        default:
            isArchSupported = false;
            break;
        }
        isOpenJCEPlusSupported = isOsSupported && isArchSupported;
```